port/include/input.h

```diff
@@ -232,6 +274,100 @@ void inputMouseSetSpeed(f32 x, f32 y);
 s32 inputMouseIsEnabled(void);
 void inputMouseEnable(s32 enabled);
 
+// Gyro Controller Initialization & Management
+void inputHandleGyroController(s32 cidx);
+void closeGyroController(s32 cidx);
+s32 inputGyroIsEnabled(s32 cidx);
+void inputGyroEnable(s32 cidx, s32 enabled);
+
+// Gyro Vertical/Horizon Mixer Adjustment
+f32 inputGetGyroVHMixer(s32 cidx);
+void inputSetGyroVHMixer(s32 cidx, f32 value);
+
+// Raw Gyro Movement Retrieval
+void inputGyroGetRawDelta(s32 cidx, s32* dx, s32* dy, s32* dz);
+
+// Update Gyro Data
+void inputUpdateGyro(s32 cidx);
+
+// Scaled Gyro Movement Retrieval
+void inputGyroGetScaledDelta(s32 cidx, f32* dx, f32* dy, f32* dz);
+
+// Gyro Crosshair Movement Retrieval & Adjustment
+void inputGyroGetScaledDeltaCrosshair(s32 cidx, f32* dx, f32* dy);
+void inputGyroGetCrosshairDelta(s32 cidx, f32* dx, f32* dy);
+
+// Gyro FreeLook Sensitivity Management
+void inputGyroGetSpeed(s32 cidx, f32* x, f32* y);
+void inputGyroSetSpeed(s32 cidx, f32 x, f32 y);
+f32 inputGyroGetSpeedX(s32 cidx);
+void inputGyroSetSpeedX(s32 cidx, f32 x);
+f32 inputGyroGetSpeedY(s32 cidx);
+void inputGyroSetSpeedY(s32 cidx, f32 y);
+
+// Gyro Aim Mode Sensitivity Management
+void inputGyroGetAimSpeed(s32 cidx, f32* x, f32* y);
+void inputGyroSetAimSpeed(s32 cidx, f32 x, f32 y);
+f32 inputGyroGetAimSpeedX(s32 cidx);
+void inputGyroSetAimSpeedX(s32 cidx, f32 x);
+f32 inputGyroGetAimSpeedY(s32 cidx);
+void inputGyroSetAimSpeedY(s32 cidx, f32 y);
+
+// Gyro Aim Mode Management
+s32 inputGetGyroAimMode(s32 cidx);
+void inputSetGyroAimMode(s32 cidx, s32 mode);
+void applyGyroAimMode(s32 cidx, f32* deltaX, f32* deltaY, f32* deltaZ);
+
+// Gyro Axis Mapping Management
+enum gyroaxismode inputGetGyroAxisMode(s32 cidx);
+void inputSetGyroAxisMode(s32 cidx, enum gyroaxismode mode);
+void applyGyroAxisMapping(s32 cidx, float gyroData[3], float accelData[3], f32* deltaX, f32* deltaY, f32* deltaZ);
+
+// Accelerometer Raw Data Access
+void inputAccelGetRawDelta(s32 cidx, f32* dx, f32* dy, f32* dz);
+
+// Gyro Activation Mode Management
+s32 inputGetGyroModifier(s32 cidx);
+void inputSetGyroModifier(s32 cidx, s32 mode);
+void applyGyroModifier(f32* deltaX, f32* deltaY, f32* deltaZ, s32 activationMode, s32 idx);
```

Comment:
Functions that are only used in input.c should be made static and not declared in input.h.